dist/s_docs

```diff
@@ -4,6 +4,10 @@ t=__wt.$$
 t2=__wt.$$.2
 trap 'rm -f $t $t2' 0 1 2 3 13 15
 
+# Insulate against locale-specific sort order
+LC_ALL=C
```

Comment:
Could just do in one line:
```bash
export LC_ALL=C
```

Reply:
I agree. Just trying to follow the existing style (used in s_string and s_typedef).